src/Phantoman.php

```diff
@@ -248,9 +238,30 @@ private function isWindows()
         return strtoupper(substr(PHP_OS, 0, 3)) === 'WIN';
     }
 
-    // methods that handle events
-    public function beforeSuite(\Codeception\Event\SuiteEvent $e)
+
+
+    public function afterSuite(\Codeception\Event\SuiteEvent $e)
+    {
+        if ($this->resource) {
+            proc_terminate($this->resource);
```

Comment:
I think we should take out the `afterSuite` since this will try to kill PhantomJS after every  suite. So if there's more than one suite needing PhantomJS, it'd get killed after the first.

The `stopServer` call in `__destruct` should be sufficient.